app/views/auctions/_bid_button.html.erb

```diff
@@ -1 +1,3 @@
-<a href="<%= new_auction_bid_path(auction) %>" class='usa-button usa-button-outline-inverse button-bid'>Bid <icon class="fa fa-angle-double-right"></icon></a>
+<%= link_to 'Bid <icon class="fa fa-angle-double-right"></icon>'.html_safe,
```

Comment:
I generally am in favor of using link_to, but this seemed like an edge case, and you have to do some unconventional stuff to make it work. I'm fine with this, but is there a good rule on when to use which?